### DISCUSSION TITLE
termux installation help

### DISCUSSION THREAD

**Discussion by Jasssg** [termux/termux-app#2868] [Q&A] [answered]
i tried to install termux .118'after setting up all things, when i entered chmod+x.... command nothing happens
if i entered super user command then it ask for root, kindly assist me![Screenshot_20220706_101719.jpg](https://user-images.githubusercontent.com/108751467/177481499-6f77342d-e0b6-41f1-85bc-2b22f543b9ac.jpg)![Screenshot_20220706_102626.jpg](https://user-images.githubusercontent.com/108751467/177481558-19983621-d3df-44f9-9063-6a237c8f7094.jpg)

**Comment by sylirre** [comment, marked as answer]
`su` or `sudo` works only on rooted device. In Termux it is only a shell script wrapper for system `su` binary which is usually installed by Magisk, SuperSU or other rooting method.

Also, in Termux you should never run `apt` under `sudo`.

**Reply by Jasssg** [reply]
sir, but chmod +x ....... command do nothing, how can i proceed

**Reply by 2096779623** [reply]
https://wiki.termux.com/wiki/hacking

**Reply by Jasssg** [reply]
Sir i tried to install termux packages first time, when i entered chmod+x ..... Command to execute, it did nothing, plz assist me what is the problem and how can i fix it

**Reply by sylirre** [reply]
`chmod +x` just sets executable bit. It won't print output to console.
